src/docs/07-operation-guide/01-setup.md

```diff
@@ -0,0 +1,346 @@
+| history.shardSyncMinInterval | the minimal time interval which the shard info should be sync to remote | 5*time.Minute|
+| history.shardSyncMinInterval | the sync shard jitter coefficient | #N/A|
+| history.defaultEventEncoding | the encoding type for history events | string(common.EncodingTypeThriftRW)|
+| history.numArchiveSystemWorkflows | key for number of archive system workflows running in total | 1000|
+| history.archiveRequestRPS | the rate limit on the number of archive request per second | 300 // should be much smaller than frontend RPS|
+| history.enableAdminProtection | whether to enable admin checking | FALSE|
+| history.adminOperationToken | the token to pass admin checking | common.DefaultAdminOperationToken|
+| history.historyMaxAutoResetPoints | the key for max number of auto reset points stored in mutableState | DefaultHistoryMaxAutoResetPoints|
+| history.enableParentClosePolicy | whether to ParentClosePolicy | TRUE|
+| history.parentClosePolicyThreshold | decides that parent close policy will be processed by sys workers(if enabled) ifthe number of children greater than or equal to this threshold | 10|
+| history.numParentClosePolicySystemWorkflows | key for number of parentClosePolicy system workflows running in total | 10|
+| history.throttledLogRPS | the rate limit on number of log messages emitted per second for throttled logger | 4|
+| history.stickyTTL | to expire a sticky tasklist if no update more than this duration | time.Hour*24*365|
+| history.decisionHeartbeatTimeout | for decision heartbeat | time.Minute*30|
+| history.DropStuckTaskByDomain | whether stuck timer/transfer task should be dropped for a domain | FALSE|
+
+
+### Dynamic Configuration for System Worker Service
+
+| Config Key | Explanation | Default Values |
+| ---------- | ----------- | -------------- |
+| worker.persistenceMaxQPS | the max qps worker host can query DB | 500|
+| worker.persistenceGlobalMaxQPS | the max qps worker cluster can query DB | 0|
+| worker.replicatorMetaTaskConcurrency | the number of coroutine handling metadata related tasks | #N/A|
+| worker.replicatorTaskConcurrency | the number of coroutine handling non metadata related tasks | #N/A|
+| worker.replicatorMessageConcurrency | the max concurrent tasks provided by messaging client | #N/A|
+| worker.replicatorActivityBufferRetryCount | the retry attempt when encounter retry error on activity | #N/A|
+| worker.replicatorHistoryBufferRetryCount | the retry attempt when encounter retry error on history | #N/A|
+| worker.replicationTaskMaxRetryCount | the max retry count for any task | #N/A|
+| worker.replicationTaskMaxRetryDuration | the max retry duration for any task | #N/A|
+| worker.replicationTaskContextDuration | the context timeout for apply replication tasks | #N/A|
+| worker.workerReReplicationContextTimeout | the context timeout for end to end re-replication process | #N/A|
+| worker.enableReplication | the feature flag for kafka replication | #N/A|
```

Comment:
Those are deprecated.

Reply:
deleted

Reply:
also post a PR to remove the config from the code base